packages/meteor-wallet/src/lib/meteor-wallet.spec.ts

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable @nrwl/nx/enforce-module-boundaries */
```

Comment:
Let's just shelve this until we have Meteor sorted out. No value in adding an empty test at the moment